### PR TITLE
Polish the README

### PR DESCRIPTION
**Pull request by oliverklee**
- fix some typos
- clean up the CSS and PHP examples
- clean up the Markdown formatting
- recommend `composer install` for installing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -611,20 +661,30 @@ class Sabberworm\CSS\CSSList\Document#4 (2) {
 
 ## Contributors/Thanks to
```
Comment (sabberworm) [inline README.md:662]: I don’t like breaking up list items over multiple lines. Makes it hard to move them up or down in the editor (which is something I tend to do a lot more often with list items than with other paragraphs).

File: README.md [hunk 2]
```diff
@@ -611,20 +661,30 @@ class Sabberworm\CSS\CSSList\Document#4 (2) {
...
```
Comment (sabberworm) [inline README.md:663]: You could add yourself here:

```suggestion

 * [oliverklee](https://github.com/oliverklee) for lots of refactorings, code modernizations and CI integrations
```

File: README.md (hunk 1 above)
Comment (sabberworm) [reply README.md:662]: Come to think of it, I also prefer to rely on the editor’s soft-wrap behaviour for paragraphs. But my opinions on that are not as strong as on list items.

**Review by sabberworm — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (oliverklee) [reply README.md:613]: Rebased, changed and repushed.

**Review by sabberworm — APPROVED** [review]

**Pull request merged**